### DISCUSSION TITLE
Call for community insights: Applications of Quarkus on AI with High Performance Edge computing

### DISCUSSION THREAD

**Discussion by PetoMichalak** [quarkusio/quarkus#35466] [Community]
Hello,

I'm working at the [National Innovation Centre for Data](https://nicd.org.uk/) with two of my colleagues examining Federated AI Learning for image classification on high performance edge computing platforms (Jetson family boards). 

We would like to hear from the Quarkus communities about your possible needs around Edge computing and AI specifically with relation to the Quarkus ecosystem. We would also like to know what AI techniques you are examining, and what Edge computing issues have you encountered, or you anticipate facing.

We invite you to comment on this thread to share your ideas or experiences of using Quarkus on Edge for AI applications which could help shape the direction of our exploration.

Thank you.

Peter Michalák
Edge Solution Specialist

**Comment by StuartWheater** [comment]
Hi,
    Has anyone used the Quarkus integration provided by Deep Java Library (DJL)?

Stuart

**Reply by maxandersen** [reply]
I haven't tried it but I've seen they mention they have Quarkus support. 

Couldn't find it though :/

**Reply by StuartWheater** [reply]
The only documentation for DJL's Quarkus extension I have seen is [https://docs.djl.ai/docs/demos/quarkus/extension/index.html](https://docs.djl.ai/docs/demos/quarkus/extension/index.html).
I wonder if it would be more used if it was included in the Quarkiverse.
